internal/versions/versions.go

```diff
@@ -307,11 +308,28 @@ func (h *handler) GetLatestOsImage(cpuArchitecture string) (*models.OsImage, err
 		}
 	}
 	if latest == nil {
-		return nil, errors.Errorf("No OS images are available")
+		return nil, errors.Errorf("No OS images are available for the specified architecture %s", cpuArchitecture)
```

Comment:
Nit, but I don't think this should be changed in this PR.

Reply:
Removed this change, thank you!